fiatconnect-api.md

```diff
@@ -2233,6 +2234,40 @@ A KYC schema containing personal data about a user, as well as documents such as
 
 The `selfieDocument` and `identificationDocument` fields should be base64 encoded binary blobs representing images.
 
+#### 9.3.1.2. `PersonalDataAndDocumentsWithBack`
+
+A brasilian standard KYC schema containing personal data about a user, as well as documents such as an ID photo and selfie.
+
+```
+{
+	fullName: `string`,
+	dateOfBirth: {
+		day: `string`,
+		month: `string`,
+		year: `string`
+	},
+	address: {
+		address1: `string`,
+		address2?: `string`,
+		isoCountryCode: `string`,
+		isoRegionCode: `string`,
+		city: `string`,
+		postalCode?: `string`
+	},
+	phoneNumber: `string`,
+	idNumber: `string`,
+	email: `string`,
+	selfieDocument: `string`,
+	identificationDocumentFront: `string`,
+	identificationDocumentBack: `string`,
```

Comment:
what kind of identity document is accepted? Any? Or just CPF cards? The description of the schema should probably go into this-- we're currently running into trouble with another KYC schema due to the ambiguity there on what document types are acceptable.

If it's CPF only, the schema should probably also be renamed.

Reply:
@cajubelt the docs are those accepted as an official identification document in Brazil: `RG`, `CNH`, `RNM`